.gitignore

```diff
@@ -1,3 +1,4 @@
 .aws-sam
 lambda.zip
 main
+venv
```

Comment:
If you're installing aws sam via pip it's good to do it in a python virtual environment:
https://cloud.google.com/python/setup